test/integration/user_research_recruitment_banner_test.rb

```diff
@@ -0,0 +1,38 @@
+require "test_helper"
+
+class UserResearchRecruitmentBannerTest < ActionDispatch::IntegrationTest
+  should "not display the banner on the login page" do
+    visit new_user_session_path
+
+    assert_not has_content?(user_research_recruitment_banner_title)
+  end
+
+  should "display the banner on the dashboard" do
+    user = create(:user, name: "user-name", email: "user@example.com")
+    visit new_user_session_path
+    signin_with(user)
+
+    assert has_content?(user_research_recruitment_banner_title)
+    assert has_link?("Sign up to take part in research", href: "https://docs.google.com/forms/d/1Bdu_GqOrSR4j6mbuzXkFTQg6FRktRMQc8Y-q879Mny8/viewform")
+  end
+
+  should "not display the banner on any page other than the dashboard" do
+    user = create(:user, name: "user-name", email: "user@example.com")
+    visit new_user_session_path
+    signin_with(user)
+
+    click_on "Change your email or password"
+
+    assert_not has_content?(user_research_recruitment_banner_title)
+  end
+
+  should "hide the banner until the next session"
```

Comment:
I'm re-learning MiniTest right now. I was expecting this blockless `should` to count as a skipped test in the test runner's report (a technique I'm familiar with from outside-in testing with other tools), but it doesn't. But I'm assuming that this is a fairly common approach?